modules/repository/repository-master/src/main/java/org/eclipse/dirigible/repository/master/jar/JarRepository.java

```diff
@@ -36,8 +36,8 @@ public class JarRepository extends ZipRepository {
      */
     public JarRepository(String zip) throws LocalRepositoryException, IOException {
 
-        InputStream in = JarRepository.class.getClassLoader()
-                                            .getSystemResourceAsStream(zip);
+        JarRepository.class.getClassLoader();
```

Comment:
I don't like this, why can't we use `JarRepository.class.getResourceAsStream(zip);` here? @ThuF @delchev are we using this `JarRepository`?

Reply:
fixed